src/Features/LanguageServer/Protocol/Features/UnifiedSuggestions/UnifiedSuggestedActionsSource.cs

```diff
@@ -207,9 +213,19 @@ static CodeFixGroupKey GetGroupKey(CodeFix fix)
                 return null;
             }
 
+            var document = fixAllState.Document!;
             using var fixAllSuggestedActionsDisposer = ArrayBuilder<IUnifiedSuggestedAction>.GetInstance(out var fixAllSuggestedActions);
             foreach (var scope in supportedScopes)
             {
+                if (scope is FixAllScope.ContainingMember or FixAllScope.ContainingType &&
+                    document.GetLanguageService<IFixAllSpanMappingService>() is IFixAllSpanMappingService spanMappingService)
```

Comment:
interesting.  so if we have a scope of ContainingMember/Type and the language does *not* export sucha  service, then we proceed?  should we filter those out instead?